combinations/hash.tsv

```diff
@@ -128,3 +128,4 @@ kraken2=2.1.1,pigz=2.6
 merqury=1.1,tar=1.34,findutils=4.6.0
 star=2.6.1d,samtools=1.10
 star=2.6.1d,samtools=1.10,gawk=5.1.0
+planemo=0.74.3,bioblend=0.15.0,ephemeris=0.10.6,galaxyxml=0.4.14
```

Comment:
```suggestion
planemo=0.74.3,galaxyxml=0.4.14
```
bioblend and ephemeris are direct requirements of planemo, you don't need to specify them.

Reply:
Thanks @mvdbeek. I'll look next time :)